e2e/cypress/integration/account_settings/display/message_display_mode_spec.js

```diff
@@ -12,55 +12,63 @@
 
-    ['COMPACT', 'STANDARD'].forEach((display) => {
-        it(`M14283 ${display} view: Line breaks remain intact after editing`, () => {
-            cy.uiChangeMessageDisplaySetting(display);
+    const firstLine = `First line ${getRandomId()}`;
+    const secondLine = `Text after ${getRandomId()}`;
 
-            const firstLine = `First line ${getRandomId()}`;
-            const secondLine = `Text after ${getRandomId()}`;
+    // # Enter in text
+    cy.get('#post_textbox').
+        clear().
+        type(firstLine).
+        type('{shift}{enter}{enter}').
+        type(`${secondLine}{enter}`);
 
-            // # Enter in text
-            cy.get('#post_textbox').
-                clear().
-                type(firstLine).
-                type('{shift}{enter}{enter}').
-                type(`${secondLine}{enter}`);
+    // # Get last postId
+    cy.getLastPostId().then((postId) => {
+        const postMessageTextId = `#postMessageText_${postId}`;
 
-            // # Get last postId
-            cy.getLastPostId().then((postId) => {
-                const postMessageTextId = `#postMessageText_${postId}`;
+        // * Verify HTML still includes new line
+        cy.get(postMessageTextId).should('have.html', `<p>${firstLine}</p>\n<p>${secondLine}</p>`);
 
-                // * Verify HTML still includes new line
-                cy.get(postMessageTextId).should('have.html', `<p>${firstLine}</p>\n<p>${secondLine}</p>`);
+        // # click dot menu button
+        cy.clickPostDotMenu(postId);
 
-                // # click dot menu button
-                cy.clickPostDotMenu(postId);
+        // # click edit post
+        cy.get(`#edit_post_${postId}`).scrollIntoView().should('be.visible').click();
 
-                // # click edit post
-                cy.get(`#edit_post_${postId}`).scrollIntoView().should('be.visible').click();
+        // # Add ",edited" to the text
+        cy.get('#edit_textbox').type(',edited');
 
-                // # Add ",edited" to the text
-                cy.get('#edit_textbox').type(',edited');
+        // # Save
+        cy.get('#editButton').click();
 
-                // # Save
-                cy.get('#editButton').click();
+        // * Verify HTML includes newline and the edit
+        cy.get(postMessageTextId).should('have.html', `<p>${firstLine}</p>\n<p>${secondLine},edited</p>`);
 
-                // * Verify HTML includes newline and the edit
-                cy.get(postMessageTextId).should('have.html', `<p>${firstLine}</p>\n<p>${secondLine},edited</p>`);
+        // * Post should have (edited)
+        cy.get(`#postEdited_${postId}`).
+            should('be.visible').
+            should('contain', '(edited)');
+    });
+}
 
-                // * Post should have (edited)
-                cy.get(`#postEdited_${postId}`).
-                    should('be.visible').
-                    should('contain', '(edited)');
-            });
+describe('Account Settings', () => {
```

Comment:
I don't think we've documented it but we typically placed the `describe` block on top so that it's immediately seen when you open the file.
Typical arrangement would be:
```
import...

describe () {
    // variables that are shared among `it` blocks
    const
    let

    before/after()

    it()
}

// helper functions
function() {}
```

Reply:
Thanks for catching it @saturninoabril! That was an oversight in a rush to push the changes through :) Fixed!